uber/site_sections/jobs.py

```diff
@@ -224,3 +224,36 @@ def add_volunteers_by_dept(self, session, message='', location=None):
                                 .order_by(Attendee.full_name).all()
             ]
         }
+
+    @ajax
+    def add_bulk_admin_accounts(self, session, ids, message='', **params):
+        if isinstance(ids, str):
+            ids = str(ids).split(",")
+        success_count = 0
+        for id in ids:
+            try:
+                uuid.UUID(id)
+            except ValueError:
+                pass
+            else:
+                match = session.query(Attendee).filter(Attendee.id == id).first()
+                if match:
+                    account = session.admin_account(params, checkgroups=['access'])
+                    if account.is_new:
+                        password = genpasswd()
+                        account.hashed = bcrypt.hashpw(password, bcrypt.gensalt())
+                        account.attendee = match
+                        session.add(account)
+                        body = render('emails/accounts/new_account.txt', {
+                            'account': account,
+                            'password': password
+                        })
+                        send_email(c.ADMIN_EMAIL, match.email, 'New ' + c.EVENT_NAME + ' Ubersystem Account', body)
```

Comment:
update function would end up going back to accounts page. I decided to rebuild it instead.